api/HTMLMarqueeElement.json

```diff
@@ -444,7 +444,7 @@
               "version_added": "65"
             },
             "ie": {
-              "version_added": "4"
+              "version_added": false
```

Comment:
Another capitalization mismatch. @vinyldarkscratch can you tweak find-missing.js a bit to find all entries which have multiple capitalization variants and ensure that only valid and known cases like `defaultstatus` + `defaultStatus` remain?